services/src/atdd/src/test/java/org/openkilda/atdd/NorthboundRunTest.java

```diff
@@ -41,17 +37,26 @@
 import org.openkilda.northbound.dto.switches.RulesSyncResult;
 import org.openkilda.northbound.dto.switches.RulesValidationResult;
 
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class NorthboundRunTest {
     private static final FlowState expectedFlowStatus = FlowState.UP;
-    private static final PathInfoData expectedFlowPath = new PathInfoData(0L, Arrays.asList(
+    private static final PathInfoData expectedForwardFlowPath = new PathInfoData(0L, Arrays.asList(
             new PathNode("de:ad:be:ef:00:00:00:03", 2, 0),
             new PathNode("de:ad:be:ef:00:00:00:04", 1, 1),
             new PathNode("de:ad:be:ef:00:00:00:04", 2, 2),
             new PathNode("de:ad:be:ef:00:00:00:05", 1, 3)));
+    private static final PathInfoData expectedReverseFlowPath = new PathInfoData(0L, Arrays.asList(
+            new PathNode("de:ad:be:ef:00:00:00:05", 1, 0),
+            new PathNode("de:ad:be:ef:00:00:00:04", 2, 1),
+            new PathNode("de:ad:be:ef:00:00:00:04", 1, 3),
+            new PathNode("de:ad:be:ef:00:00:00:03", 2, 3)));
```

Comment:
why seqIds (3d param) are 0-1-3-3 for reverse path in PathNode? I would expect it to be 0-1-2-3

Reply:
Misspelling from my side. But this field is excluded from PathNode's equals check, is it correct?

Reply:
Dunno. Most likely it is not actually used anywhere, but I'm not sure

Reply:
Have tried to replace boiler plate code in PathNode with `@Data` and had failing tests in PathComputatationEngine. So fix misspelling and leave it as is